src/main/java/ch/ejpd/lgs/searchindex/client/service/processor/PersonDataProcessor.java

```diff
@@ -42,7 +42,7 @@ public void listenPartial(PersonData personData, @Headers Map<String, Object> ra
     try {
       out(
           Topics.PERSONDATA_PARTIAL_OUTGOING,
-          process(personData, headers.getSenderId()),
+          process(personData, headers.getSenderId(), null),
```

Comment:
why is the landregister not considered here?

Reply:
This is done for partial sync. The land registers are used/implemented only for full sync

Reply:
How is the landregister handled in the partial sync?

Reply:
the data must still arrive on LGS-side with the landregister info